### PR TITLE
Fixed proto file.

### PR DESCRIPTION
**Pull request by normanjaeckel**
@peb-adr Please check this again. The syntax of the current main branch proto file is definitly wrong. There is a `>` missing. So I tried the following fix.

### CODE REVIEW

**Comment by normanjaeckel** [comment]
See #157

File: proto/manage.proto [hunk 1]
```diff
@@ -34,7 +34,7 @@ message CreateUserRequest {
     string default_password = 5 [ json_name = "default_password" ];
     string email = 6 [ json_name = "email" ];
     string organization_management_level = 7 [ json_name = "organization_management_level" ];
-    map<string,  committee__management_level = 8 [ json_name = "committee__management_level" ];
+    map<string, google.protobuf.ListValue> committee__management_level = 8 [ json_name = "committee__management_level" ];
```
Comment (peb-adr) [inline proto/manage.proto:37]: Wow, this is exactly what i wanted to do.
Honestly I'm quite ashamed rn I didn't notice commiting the wrong thing ...

**Review by peb-adr — APPROVED** [review]
Tested this, works.
Thanks for working this out. You can merge it when you want.

**Comment by peb-adr** [comment]
@normanjaeckel

File: proto/manage.proto (hunk 1 above)
Comment (normanjaeckel) [reply proto/manage.proto:37]: ;-)

**Pull request merged**